src/test/java/org/fundacionjala/coding/Gary/TestCheckSum.java

```diff
@@ -0,0 +1,60 @@
+package org.fundacionjala.coding.Gary; /**
+ * Created by Gary on 29/8/2017.
+ */
+
+
+import org.junit.Before;
+import org.junit.Test;
+
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+/**
+ * TestCheckSum.
+ */
+public class TestCheckSum {
```

Comment:
to increase the code coverage you should test when the checksum is 0.
for instance.
```java
 assertTrue(checkSum.checkSum1("9783827317100"));
```

Reply:
You need to test this scenario